CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Package-specific changes not released in any SDK will be added here just before
 
 ### 📚 3rd party library updates
 
+- Removed `react-native-appearance` that is deprecated since SDK 43. Migrate to [`Appearance` API](https://reactnative.dev/docs/appearance). ([#16436](https://github.com/expo/expo/pull/16436) by [@kudo](https://github.com/kudo))
```

Comment:
Isn't this a breaking change, instead of a `3rd party library update`?

Reply:
that's a good point. i will update accordingly

Reply:
added to release notes and updated changelog

Reply:
good catch, thanks!